classes/includes/src/Services/Gateway/HpsSoapGatewayService.php

```diff
@@ -38,7 +38,9 @@ public function doRequest($transaction, $options = array())
         if ($this->_config->developerId != null && $this->_config->developerId != "") {
             $hpsHeader->appendChild($xml->createElement('hps:DeveloperID', $this->_config->developerId));
             $hpsHeader->appendChild($xml->createElement('hps:VersionNbr', $this->_config->versionNumber));
-            $hpsHeader->appendChild($xml->createElement('hps:SiteTrace', $this->_config->siteTrace));
+            if(!empty($this->_config->siteTrace)) {
```

Comment:
Why are we adding this in? I don't think the plugin will ever pass in a value here.

Reply:
I have removed this condition.